### ISSUE TITLE
[Fix]: Accordion custom styles can be removed

### ISSUE THREAD

**Issue by castastrophe** [adobe/spectrum-web-components#3059]
As the referenced issue [#1033](https://github.com/adobe/spectrum-css/issues/1033) has been merged and released, should this custom code now be removed? If you approve, I am happy to open said PR.

Link: [accordion-item.css](https://github.com/adobe/spectrum-web-components/blob/d45dffddd23c16611cb9c0fa645e2a2cb020bb95/packages/accordion/src/accordion-item.css#L19-L32)

```js
/**
 * What follows are styles working around: https://github.com/adobe/spectrum-css/issues/1033
 */
#indicator {
    /* .spectrum-Accordion-itemIndicator */
    top: calc(
        50% -
            var(
                --spectrum-accordion-icon-height,
                var(--spectrum-global-dimension-size-125)
            ) / 2
    );
}

...
```

**Comment by Rajdeepc** [comment]
@Westbrook I think we can close this issue.

**Comment by Westbrook** [comment]
fixed in #3290
